pkg/controller/controller.go

```diff
@@ -132,10 +132,10 @@ func (c AppGwIngressController) Process(event QueuedEvent) error {
 	if err != nil {
 		// Reset cache
 		c.configCache = nil
-		glog.Warningf("unable to send CreateOrUpdate request, error [%v]", err.Error())
 		configJSON, _ := c.dumpSanitizedJSON(&appGw)
-		glog.V(5).Info(string(configJSON))
-		return errors.New("unable to send CreateOrUpdate request")
+		glog.Errorf("Failed applying App Gwy configuration: %s -- %s", err, string(configJSON))
+		// TODO(draychev) -- c.recorder.Event() - what object?
```

Comment:
Good Question: but it would hard to point out which ingress caused this.
As such, it would be easier to log event to all the ingresses saying that AppGW Update failed with so an so reason.

In a good world, our valdations on controller to should be good enough that this doesn't happen.

Reply:
You're right. This might not make sense here.
I removed the comment.